backend_new/src/controllers/ami-chart.controller.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put,
+  Query,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import {
+  ApiExtraModels,
+  ApiOkResponse,
+  ApiOperation,
+  ApiTags,
+} from '@nestjs/swagger';
+import { AmiChartService } from '../services/ami-chart.service';
+import { AmiChart } from '../dtos/ami-charts/ami-chart-get.dto';
+import { AmiChartCreate } from '../dtos/ami-charts/ami-chart-create.dto';
+import { AmiChartUpdate } from '../dtos/ami-charts/ami-chart-update.dto';
+import { defaultValidationPipeOptions } from '../utilities/default-validation-pipe-options';
+import { AmiChartQueryParams } from '../dtos/ami-charts/ami-chart-query-params.dto';
+import { IdDTO } from '../dtos/shared/id.dto';
+import { SuccessDTO } from '../dtos/shared/success.dto';
+
+@Controller('/amiCharts')
+@ApiTags('amiCharts')
+@UsePipes(new ValidationPipe(defaultValidationPipeOptions))
+@ApiExtraModels(AmiChartCreate, AmiChartUpdate, IdDTO, AmiChartQueryParams)
```

Comment:
I know we have plans to update access control, but I think we should still add the guards we have in the mean time. Also please update the tests to include testing against someone who does not have permissions for CUD.

Reply:
Hey Sean I think we should add guards/permissioning in a separate ticket since most of our operations require a logged in user and logging in is being handled in a later ticket 

adding the guards would more or less mean that nothing could be tested until that happens